result/dhcpv4serverleases.yaml

```diff
@@ -0,0 +1,81 @@
+components:
+  schemas:
+    Dhcpv4ServerLease.State.Request:
+      description: >-
+        The request to retrieve DHCP Server host allocated status.
+      type: object
+      properties:
+        dhcp_server_names:
+          description: >-
+            The names of DHCPv4 server to return results for.
+            An empty list will return results for all DHCPv4 servers.
+          type: array
+          items:
+            type: string
+          x-constraint:
+          - "/components/schemas/Device.Dhcpv4server/properties/name"
+          x-field-uid: 1     
+    Dhcpv4ServerLeases.State:
+      description: >-
+       Lease information of DHCP Server.
+      type: object
+      properties:
+        dhcp_client_name:
+          description: >-
+            The name of a DHCP Server.
+          type: string
+          x-field-uid: 1
+        leases:
+          type: array
+          items:
+            $ref: '#/components/schemas/Dhcpv4ServerLease.State'
+          x-field-uid: 2
+    Dhcpv4ServerLease.State:
+      description: >-
+        IPv4 unicast prefix.
+      type: object
+      properties:
+        address:
+          description: >-
+            The IPv4 address associated with this lease.
+          type: string
+          x-field-uid: 1
+        valid_time:
+          description: >-
+            The time in seconds, IP address lease will expire.
+          type: integer
+          format: uint32
+          x-field-uid: 2
+        preferred_time:
+          description: >-
+            The time in seconds, elapsed time since address has been renewed.
```

Comment:
'The elapsed time in seconds since the address has been renewed. ' might be better.